packages/create-rslib/fragments/base/react-ts/tsconfig.json

```diff
@@ -10,5 +10,5 @@
     "moduleResolution": "bundler",
     "useDefineForClassFields": true
   },
-  "include": ["src"]
+  "exclude": ["node_modules"]
```

Comment:
Will this change affect the dts results? The best practice should be that the outer `tsconfig.json` is responsible for all type verification, and there is a `tsconfig.build.json` responsible for emitting dts files.

Reply:
Agreed, and, without using project reference?

Reply:
You're right, dts messed up, a lot of config file emitted. 🥲

Reply:
`the test files are also included and get correct type resolve`  I think this should configure in user side. Our tpl can ignore this.